api/src/routes/clubs.ts

```diff
@@ -0,0 +1,94 @@
+import express from "express";
+import type { Response, Request } from "express";
+import prisma from "../prisma_client.ts";
+
+const router = express.Router();
+
```

Comment:
```suggestion
router.use(express.json())
```
needs to be added so that the request can parse the JSON data being sent.

Reply:
added